sfa_dash/templates/sections/notifications.html

```diff
@@ -1,29 +1,38 @@
-{% if messages is defined %}
```

Comment:
Not critical, but it would be nice not to have the empty divs floating around. Maybe something like
```
{% set flashed_messages = get_flashed_messages(category_filter=['messages']) %}
{% set flashed_warnings ...%}
{% if messages is defined or (flash_messages | length) > 0 %}
<div class="messages">
  <ul class="message-list notifcation-list">
    {% if messages is defined %} # still need this for now
...
```